Doc/library/io.rst

```diff
@@ -739,8 +740,8 @@ than raw I/O does.
 
 .. class:: BufferedRandom(raw, buffer_size=DEFAULT_BUFFER_SIZE)
 
-   A buffered interface to random access streams.  It inherits
-   :class:`BufferedReader` and :class:`BufferedWriter`.
+   A buffered binary stream over a seekable :class:`RawIOBase` raw binary stream.
```

Comment:
Can you link to the `seekable()` method here?

Reply:
Alright.

Reply:
Ok, that makes sense. Is the adjective "seekable" clear enough to the reader that it means that the class returns true on `seekable()`. If so, we're good -- if not, perhaps it would be good to add a sentence clarifying that act.

Reply:
Ok, that makes sense. Is it clear enough to the reader that the adjective "seekable" means that the class returns True when calling `seekable()`? If so, we're good -- if not, perhaps it would be good to add a sentence clarifying that fact.

Reply:
Actually "seekable" was already used in the next paragraph, so I think we are safe.